README.md

```diff
@@ -23,7 +23,9 @@ To start the main services excluding the example typescript template app:
 
 `docker-compose up --scale=app=0`
 
-Install dependencies using `npm install`, ensuring you are using >= `Node v14.x`
+Install dependencies using `npm install`, ensuring you are using `Node v18.12.x`
+
+If using `nvm` (encouraged - `brew install nvm` or see [nvm project](https://github.com/nvm-sh/nvm#about)), then run `nvm use` within the repository folder to use the correct version.
```

Comment:
pro tip: there's an nvm alternative which _infinitely_ faster if you use zsh on a mac (maybe on other oses too) where nvm is really really annoying
https://github.com/Schniz/fnm

Reply:
I like the sound of "infinitely faster" 😄, thank you.